### PR TITLE
Add missing mdl dependencies

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

**Comment by pyhalov** [comment]
I know nothing about ruby, but these changes seems to be enough to fix our mdl.  Without this it fails with lots of missing dependencies.

File: components/ruby/mdl/Makefile [hunk 1]
```diff
@@ -15,6 +15,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		mdl
 COMPONENT_VERSION=	0.4.0
+COMPONENT_REVISION=	1
 COMPONENT_SUMMARY=	Markdown lint tool
```
Comment (AndWac) [inline components/ruby/mdl/Makefile:19]: I think you'll need to put this in a separate PR.

Comment (pyhalov) [reply components/ruby/mdl/Makefile:19]: Why?

Comment (AndWac) [reply components/ruby/mdl/Makefile:19]: Because tt requires the newly introduced packages:

depend type=require fmri=library/ruby/kramdown
depend type=require fmri=library/ruby/kramdown-parser-gfm
depend type=require fmri=library/ruby/mixlib-cli
depend type=require fmri=library/ruby/mixlib-config

Comment (pyhalov) [reply components/ruby/mdl/Makefile:19]: No, as it's runtime dependency, not build-time.

**Review by AndWac — APPROVED** [review]
